src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -1772,7 +1772,6 @@ class CreateStatementSegment(BaseSegment):
             Sequence("NOTIFICATION", "INTEGRATION"),
             Sequence("SECURITY", "INTEGRATION"),
             Sequence("STORAGE", "INTEGRATION"),
-            "VIEW",
             Sequence("MATERIALIZED", "VIEW"),
             Sequence("SECURE", "VIEW"),
```

Comment:
Pretty sure this is covered by your create view statement now so can be removed?

Reply:
Removed: https://github.com/sqlfluff/sqlfluff/pull/2217/files#diff-a8da7f7dc8f0e448950f6d5c2cd6cd5590026ea6da32cdc8f233e037bea947ecL1777